src/som/primitives/actors/PromisePrims.java

```diff
@@ -148,11 +224,11 @@ protected static final SPromise registerOnError(final SPromise rcvr,
       assert block.getMethod().getNumberOfArguments() == 2;
 
       Actor current = EventualMessage.getActorCurrentMessageIsExecutionOn();
-      SPromise  promise  = SPromise.createPromise(current);
+      SPromise  promise  = SPromise.createPromise(current, false, false, false);
```

Comment:
Right now there is no implementation for putting breakpoints on `onError` primitive and `actorsWhen:resolved:onError`, that's why I set leave them with false value, I will include them